docs/reports/requirements/2_dfd-desc.md

```diff
@@ -1,9 +1,10 @@
 # Data Flow Diagrams
 
-Section 2 of the Requirements Report
 
-An overview of your system architecture presented as a data flow diagram (DFD), at both levels 0 and level 1, along with accompanying descriptions for each diagram. You must be very specific in this DFD so to design it in a way that supports an incremental and iterative development process. You can read an intro on DFDs and adopt its notation. Remember that a process does something, so be sure to use a verb to label your processes. In describing your DFD, highlight clearly which components will be delivered for each of the following milestones: peer testing #1, peer testing #2, and the final product. You may additionally wish to include extra components and label them as bonus features that may be developed if time allows.
+## Level 0 DFD 
 
-Diagrams will be incorporated into the PDF report.
+![](dfd_0.png)
```

Comment:
I recommend adding in the point-form description for each level (basically what you said during the presentation). Then, we can use the point-form list this weekend when we write up the report!

Reply:
Thanks for the heads up! I have made those changes :)